### ISSUE TITLE
Enhance rendering tests

### ISSUE THREAD

**Issue by mrdrogdrog** [hedgedoc/hedgedoc#2909]
**Which part of the project should be enhanced?**
E2E-Tests for rendering libraries.

**Is your enhancement request related to a problem? Please describe.**
Yesterday we had to downgrade d3-graphviz because the new version depends on d3v6 but other libs depend on d3v5. Our tests didn't raise an alert because everything compiled successfully and even the basic e2e test that just checks if the libs does something was happy. But because these libs conflicts markmap was unable to put anything into the svg. The svg itself was created, but without content.

**Describe the solution you'd like**
Extend the e2e test that check the functionality of the lib renderings (mermaid, markmap, etc.). They should also check if some key words are somehow present in the output.

**Describe alternatives you've considered**
Check out every lib update and test this by ourself.

**Additional context**
@ErikMichelson, you're now allowed to say "told you so."

**Comment by jmallach** [comment]
This will be good because I found out the output of sequence diagrams in hd 2.0 is actually *worse* than in 1.x. Will file a separate issue about this soon.
